gpiozero/output_devices.py

```diff
@@ -503,6 +503,243 @@ def _blink_device(
             self._write(value)
             if self._blink_thread.stopping.wait(delay):
                 break
+            
+class PWMBuzzer(PWMOutputDevice):
```

Comment:
Should this be called `PWMBuzzer` or `TonalBuzzer`?

`PWMLED` works with normal LEDs but this class doesn't work with normal buzzers.

Reply:
I'm happy to go with `TonalBuzzer`. 

As an alternative, would `PWMTonalBuzzer` be acceptable? Just to make it clear that's the type of behaviour we should expect from this class?

Reply:
I'd go for `TonalBuzzer`